Sources/Sentry/SentrySystemEventBreadcrumbs.m

```diff
@@ -177,6 +181,58 @@ - (void)initScreenshotObserver
                                                  name:UIApplicationUserDidTakeScreenshotNotification
                                                object:nil];
 }
+
+- (NSNumber *_Nullable)storedTimezoneOffset
+{
+    SentryAppStateManager *appStateManager =
+        [SentryDependencyContainer sharedInstance].appStateManager;
```

Comment:
`m`: Can we please pass this in as a dependency with the constructor to improve testability.